src/components/card/card.tsx

```diff
@@ -1,11 +1,12 @@
 'use client';
+
 import { Card, Group, Image, Text, Badge } from '@mantine/core';
 
 import { CardProps } from '@types';
 
-import { ShoppingListTable } from './shopping-list-table';
+import { ShoppingListTable } from '../shopping-list-table/shopping-list-table';
```

Comment:
I personally don't like such long imports - what do you think about exporting from `index` files?

Reply:
fixed